mesonbuild/mtest.py

```diff
@@ -2152,12 +2159,15 @@ def run(options: argparse.Namespace) -> int:
 
     check_bin = None
     if options.gdb:
-        options.verbose = True
+        options.interactive = True
         if options.wrapper:
             print('Must not specify both a wrapper and gdb at the same time.')
```

Comment:
This message should be updated as well.

Reply:
Err, never mind. Misread.

Reply:
Not entirely sure what you were hinting at, does what I pushed work for you?

Reply:
The original was correct, because the `if` statement specifically checks for `options.gdb`. There should not be any reason why an interactive test could not have a wrapper, right?

Reply:
Oops, reverted to what I had originally